src/generated/java/com/prowidesoftware/swift/model/mt/mt3xx/MT380.java

```diff
@@ -1223,6 +1247,18 @@ public static SequenceB1 getSequenceB1(SwiftTagListBlock parentSequence) {
 		}
 		return s;
 	}
+    /**
+	 * Get the single occurrence of SequenceB1 delimited by 16R/16S the value of SequenceB1#START_END_16RS.
+	 * The presence of this method indicates that this sequence can occur only once according to the Standard.
+	 * @see SequenceB1#getSequenceB1(SwiftTagListBlock)
+	 * @param parentSequence a not null parent sequence to find SequenceB1 within it
+	 * @return the found sequence or an empty sequence if none is found, <em>never returns null</em>
+	 * @since 9.2
+	 */
+	@SequenceStyle(Type.GENERATED_16RS)
+	public static SequenceB1 getORDRPRTY(SwiftTagListBlock parentSequence) {
```

Comment:
rename to getSequenceORDRPRTY in all cases, meaning, add the "Sequence" word to the method name

Reply:
Ok